### ISSUE TITLE
Missing example input for landmarkID

### ISSUE THREAD

**Issue by gemmaelsom** [jvitale/une_ICT100_assignment4#2]
The readme references robotID and landmarkID. For robotID, it provides an example, like 'pink'.

It does not do this for landmarkID. You have to look at solution-task-2 to find an example to complete solution-task-1.

It probably makes sense to make sure the information to do a question does not require information from questions that are after that question, so we can do the questions in order.

**Comment by gemmaelsom** [comment]
I can see the readme has examples at the bottom, that's handy, I didn't see that. The documentation would be easier to read if it was consistent about where examples were referenced though (ie either having them all have an e.g. in the description or having none of them have that and having all of the examples as parameters in the example)

**Comment by jvitale** [comment]
In a documentation file the structure is normally what is currently used in this documentation: method signature, description of the method, returns values and example code. One of the objectives of this assignment is also to give the students a chance to learn how to read documentations. I added an example of landmark IDs and a link to the method that can be used to list them. The methods in the documentation are grouped by theme and are not presented in order of their use to solve the assignment. I also added a paragraph to stress the presence of an example code at the end of every method description.
